Mathlib/Data/Polynomial/Coeff.lean

```diff
@@ -3,10 +3,11 @@ Copyright (c) 2018 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Johannes Hölzl, Scott Morrison, Jens Wagemaker
 -/
-import Mathlib.Data.Polynomial.Basic
+import Mathlib.Algebra.MonoidAlgebra.Support
+import Mathlib.Algebra.Regular.Pow
 import Mathlib.Data.Finset.NatAntidiagonal
 import Mathlib.Data.Nat.Choose.Sum
-import Mathlib.Algebra.Regular.Pow
+import Mathlib.Data.Polynomial.Basic
```

Comment:
Mind checking with `#minimize_imports` that this is all needed. I'm surprised you need so much more for a single lemma.

Reply:
It seems needed. Although `Mathlib.Data.Finset.NatAntidiagonal` wasn't needed anymore, so I cut that out.